sapp/filter.py

```diff
@@ -17,7 +17,9 @@
 from .models import Base
 
 if TYPE_CHECKING:
-    from .ui.schema import FeatureCondition
+    from .ui.schema import FeatureCondition, MatchesIsField
+
+MatchesIsAlias = Union[Dict[str, Union[str, List[str]]], List[str]]
```

Comment:
I think we could use a `TypedDict` here, to better describe what that `Dict[str, Union[str, List[str]]]` looks like.

Reply:
Done